src/pages/home/report/ReportActionCompose/ComposerWithSuggestions.js

```diff
@@ -490,9 +491,13 @@ function ComposerWithSuggestions({
     }, [focus, prevIsFocused, prevIsModalVisible, isFocused, modal.isVisible, isNextModalWillOpenRef]);
 
     useEffect(() => {
+        // Set the `selection at end` and `scrolls input to bottom` for `Web Platforms`.
```

Comment:
@jeet-dhandha Could you update this comment

```suggestion
        // Scrolls the composer to the bottom and set the selection to the end
```